cordapp-cpk2/src/test/kotlin/net/corda/plugins/cpb2/CpbTest.kt

```diff
@@ -100,7 +102,8 @@ class CpbTest {
         val embeddedCpkFiles = TreeMap<String, String>()
         JarInputStream(Files.newInputStream(cpbFile), true).use { jarInputStream ->
             assertEquals("customName", jarInputStream.manifest.mainAttributes.getValue("Corda-CPB-Name"))
-            assertEquals("customVersion", jarInputStream.manifest.mainAttributes.getValue("Corda-CPB-Version"))
+            assertEquals("customVersion", jarInputStream.manifest.mainAttributes.getValue(CPB_VERSION_ATTRIBUTE))
+            assertEquals("2.0", jarInputStream.manifest.mainAttributes.getValue(CPB_FORMAT_VERSION))
```

Comment:
Any reason why you're not using `CPB_CURRENT_FORMAT_VERSION` here? Or should it be `private const`?

Reply:
I've updated the test to use the constant as the expected value.